demo/word2vec/train.py

```diff
@@ -0,0 +1,156 @@
+import math
+import pickle
+import glob
+import os
+import sys
+import paddle.v2 as paddle
+import paddle.v2.dataset.common as common
+
+embsize = 32
+hiddensize = 256
+N = 5
+
+# NOTE: You need to generate and split dataset then put it under your cloud storage.
+#       then you can use different size of embedding.
+
+# NOTE: must change this to your own username on paddlecloud.
+USERNAME = "wuyi05@baidu.com"
+DC = os.getenv("PADDLE_CLOUD_CURRENT_DATACENTER")
```

Comment:
No, for imikolov, we can't. The size of BOW is hard-coded when generating this public dataset `N=5`, it's impossible to change it if use the public dataset.